src/osemgrep/cli_scan/Core_runner.ml

```diff
@@ -106,7 +106,15 @@ let split_jobs_by_language all_rules all_targets : Lang_job.t list =
   |> Common.map_filter (fun (xlang, rules) ->
          let targets =
            all_targets
-           |> List.filter (Filter_target.filter_target_for_xlang xlang)
+           |> List.filter (fun target ->
+                  (* If the language is JS, also include TS files *)
```

Comment:
@emjin recently did some related modifications, so maybe you need to agree on where to handle this.

Reply:
will check with her

Reply:
We chatted; we'll go with mine because it'll work with my extract mode changes